lib/tasks/import_data_to_bigquery.rake

```diff
@@ -1,7 +1,28 @@
 DEFAULT_BATCH_SIZE = 200
 SLEEP_TIME = 2
+ENTITY_MODELS = %w[Candidate ApplicationChoice ApplicationExperience ApplicationFeedback
```

Comment:
Could we map these from `analytics.yml` on the scheme of https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/spec/config/analytics_spec.rb?

Reply:
Updated.